bower.json

```diff
@@ -4,19 +4,19 @@
   "homepage": "http://meanjs.org/",
   "license": "MIT",
   "dependencies": {
-    "angular": "~1.3",
-    "angular-animate": "~1.3",
+    "angular": "~1.4",
+    "angular-animate": "~1.4",
     "angular-bootstrap": "~1.0",
     "angular-file-upload": "1.1.5",
-    "angular-messages": "~1.3.17",
-    "angular-mocks": "~1.3",
-    "angular-resource": "~1.3",
+    "angular-messages": "~1.4",
+    "angular-mocks": "~1.4",
+    "angular-resource": "~1.4",
     "angular-ui-router": "~0.2",
     "angular-ui-utils": "bower",
     "bootstrap": "~3",
     "owasp-password-strength-test": "~1.3.0"
   },
   "resolutions": {
-    "angular": "~1.3"
+    "angular": "~1.4"
```

Comment:
Why is this necessary? Can we fix the dependencies so the resolution is unnecessary.